internal/rootcoord/dml_channels.go

```diff
@@ -142,7 +145,16 @@ type dmlChannels struct {
 	channelsHeap channelsHeap
 }
 
-func newDmlChannels(ctx context.Context, factory msgstream.Factory, chanNamePrefix string, chanNum int64) *dmlChannels {
```

Comment:
should not use a default channel prefix onto speciified topics, it will change exists topic to new one.